### PR TITLE
Compatibility with sass-module-importer.

### PR DESCRIPTION
**Pull request by ringods**
When using pure `npm` scripts, it is easier to use a `node-sass` importer rather than using Eyeglass. In my setup, I have [`sass-module-importer`](https://www.npmjs.com/package/sass-module-importer). This importer resolves SASS/SCSS files in node modules just by it's name, e.g.:

```
@import '@buildit/gravy'
````

As a result, the import to `typi` which is in your SASS files works too.

The SASS importer defines to pick up the main style sheet via these settings:

> Set a SCSS/Sass/CSS file on the "main" field of their package.json/bower.json
Set a SCSS/Sass/CSS file on the "style" field of their package.json/bower.json
Have a index.css file on the root of their module

I chose to add a `style` field to the `package.json` file and pointed it to the `index.scss`.

### CODE REVIEW

File: package.json
Comment (james-nash) [inline package.json:5]: Adding it as `style` is fine, but I wonder if we could just use `main` instead. If I've understood `sass-module-importer`'s README correctly, that should also work.

Looking at it now with a fresh pair of eyes, `"main": "README.md"` looks a bit odd. @ThePeach What do you think?

**Review by james-nash — COMMENTED** [review]
Thanks for the contribution @ringods!

It looks fine to me. But I'd like @ThePeach to have a quick look too. Unless having the README as our "main" entry point serves some useful purpose, I'd propose that we use that to point to our main SCSS file instead of adding an additional `style` property.

File: package.json
Comment (ringods) [reply package.json:5]: @james-nash In a combined Javascript + CSS module, the `main` property is usually used for the Javascript entrypoint, while the `style` property is used for the CSS entrypoint.

Comment (james-nash) [reply package.json:5]: Sure, but `gravy` is just a SASS package. It doesn't expose any JS. So, in this case we may as well put `main` to better use than what it's currently doing :-)

Comment (ringods) [reply package.json:5]: True. :-)

Comment (ThePeach) [reply package.json:5]: Hi @ringods and thank you for the contribution!
Just to confirm, are we going with the following?

```suggestion
  "main": "sass/index.scss",
```

I have no real preference myself, unless there's a specific technical reason for doing so, I also don't see why `main` shouldn't point to the actual "artefact" that is used. Everybody knows where the README is ;)

Aside from that, once merged I'll update the README to include sass-module-importer as I think it's a valuable resource. Thanks for sharing this.

**Comment by ThePeach** [comment]
@james-nash I think once the README has been updated (happy to do it once this has been merged) you'll have to release it. I was looking into moving into CircleCI 2.0, but this is outside my permission levels as a maintainer. Oddly enough it seems to have been configured under my name, but it's published under buildit npm account. :1234:

File: package.json
Comment (james-nash) [reply package.json:5]: Yes, I think we should. Will apply that change and get this merged.

Comment (james-nash) [reply package.json:5]: Ah, just realised I can't as it's a PR from a forked repo, not a branch on this repo. To speed things, up I'll approve and merge this and then make the amendment locally. Hope that's Ok with you both!

**Review by james-nash — APPROVED** [review]

**Pull request merged**
